src/pages/Admin/Admin.tsx

```diff
@@ -427,14 +427,14 @@ const Admin: React.FC = () => {
 
   const { isAdminUser, isLoading: isAdminUserLoading } = useIsAdminUser(userId);
 
-  const venuesOwnedByUserQuery = useMemo<ReduxFirestoreQuerySetting>(
+  const venuesOwnedByUserQuery = useMemo<SparkleRFQConfig>(
     () => ({
       collection: "venues",
       where: [["owners", "array-contains", userId]],
     }),
     [userId]
   );
-  useFirestoreConnect(venuesOwnedByUserQuery);
+  useSparkleFirestoreConnect(venuesOwnedByUserQuery);
```

Comment:
This doesn't need to be `useMemo`'d, it was a legacy assumption that proved wrong.

I would also recombine the query directly into `useSparkleFirestoreConnect` unless we need to reuse it elsewhere.